### ISSUE TITLE
Request: Add option to autostart LoFloccus with Windows

### ISSUE THREAD

**Issue by sn2020aug** [TCB13/LoFloccus#21]
I assume most/all users would want to start LoFloccus when Windows starts. It would be nice if this is an option for LoFloccus' GUI instead of adding it to the autostart folder.

**Comment by TCB13** [comment]
Yeah this is a reasonable request. I never implemented this because I’ve always went the autostart route, but yes, I’ll do it soon. Thanks for the suggestion.

**Comment by Neved4** [comment]
Would be cool to have autostart work on macOS too! 🤞🏻
